src/main/java/com/thealgorithms/sorts/RadixSort.java

```
@@ -2,61 +2,79 @@
 
 import java.util.Arrays;
 
+/**
+ * @author James Mc Dermott(theycallmemac)
+ * @see https://www.geeksforgeeks.org/radix-sort/
+ * @see https://www.youtube.com/watch?v=XiuSW_mEn7g
+ * 
+ * Radix sort is an algorithm that uses counting sort as a subroutine to sort an array of only either positive or negetive integers/strings 
+ * in either ascending or descending order.
+ * The main idea of radix sort revolves around applying counting sort digit by digit on the given array.
+ */
+
 class RadixSort {
 
-    private static int getMax(int[] arr, int n) {
-        int mx = arr[0];
-        for (int i = 1; i < n; i++) {
+    /*
+     * This method returns the maximum value in the array
+     * @param arr The array where to find the maximum value
+     */
+    private int getMax(int[] arr) {
+        int mx = Integer.MIN_VALUE;
+        for (int i = 0; i < arr.length; i++) {
```

Please throw an exception if array is empty